vnext/Microsoft.ReactNative/TurboModulesProvider.cpp

```diff
@@ -257,7 +257,9 @@ class TurboModuleImpl : public facebook::react::TurboModule {
               auto argWriter = winrt::make<JsiWriter>(runtime);
 
               // call the function
+              argWriter.WriteArrayBegin();
```

Comment:
For my work on UIManager I changed the code a few lines down to use MoveResult instead of AccessResultAsArgs.  Wouldn't that be better rather than wrapping the result in an Array and then unwrapping it?

Reply:
Let me try your approach instead.

---
In reply to: [510391001](https://github.com/microsoft/react-native-windows/pull/6304#discussion_r510391001) [](ancestors = 510391001)